package.json

```diff
@@ -9,6 +9,9 @@
     "node": "^18.14.0"
   },
   "scripts": {
+    "version:major": "yarn workspace @payment/backend version:major",
```

Comment:
We can use these later as part of a release workflow? Running these cmds and then restarting the app will update the version displayed by swagger as well as the api paths (in the case of a major version update)